src/niscope/examples/niscope_fetch_grpc.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python
+
+import argparse
+import grpc
+import niscope
+import pprint
+import sys
+
+pp = pprint.PrettyPrinter(indent=4, width=80)
+
+
+def example(resource_name, channels, options, length, voltage, address, port,):
```

Comment:
```suggestion
def example(resource_name, channels, options, length, voltage, address, port):
```
The style inconsistency of this extra comma kind of bothers me.